### PR TITLE
Update linter to react

### PR DESCRIPTION
**Pull request by 38Slava**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
       "line": { "exceptions": ["/"], "markers": ["/"] },
       "block": { "exceptions": ["*"], "markers": ["/"], "balanced": true }
     }],
-    "space-before-function-paren": ["error", { "anonymous": "never", "named": "never", "asyncArrow": "always" }]
+    "space-before-function-paren": ["error", { "anonymous": "never", "named": "never", "asyncArrow": "always" }],
+    "react/react-in-jsx-scope": 0
```
Comment (legendar) [inline .eslintrc.json]: please use `false` here

File: webpack.config.babel.js [hunk 2]
```diff
@@ -130,6 +130,7 @@ module.exports = createConfig([
         app.use(morgan('dev'))
       },
       hot: true,
+      disableHostCheck: true,
```
Comment (legendar) [inline webpack.config.babel.js]: please explain why we need this flag here ?

Comment (38Slava) [inline webpack.config.babel.js]: Explained in commit message

File: .eslintrc.json (hunk 1 above)
Comment (38Slava) [inline .eslintrc.json]: " Configuration for rule "react/react-in-jsx-scope" is invalid: Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed 'false')."
Maybe i slould use off here?

**Pull request merged**